.ado/publish.yml

```diff
@@ -55,7 +55,9 @@ extends:
          - job: RNGithubNpmJSPublish
            displayName: NPM Publish React-native-macos
            pool:
-             vmImage: $(vmImageApple)
+             name: cxeiss-ubuntu-20-04-large
+             image: cxe-ubuntu-20-04-1es-pt
```

Comment:
Just noticed that we're using a 4 year old release. Don't we have images with 22 or 24?

Reply:
Probably. I think as a follow up we can update these. I've learned the hard way that PR and CI (publish) images have to be different pools. I've generally kept the PR images up to date... but not the CI images.